students/gotem888/apple-shop/scripts/scripts.js

```diff
@@ -0,0 +1,139 @@
+import PRODUCTS from "./products.js";
+
+let searchIcon = document.getElementById("search-icon");
```

Comment:
It's better to use `const` to save a link for HTML elements.

```suggestion
const searchIcon = document.getElementById("search-icon");
```